site/content/entry/yuzu-progress-report-oct-2021/index.md

```diff
@@ -0,0 +1,430 @@
+increasing the rendering resolution while getting the full performance cost of it.
+
+{{< single-title-imgs-compare
+	"The Legend of Zelda: Link's Awakening (2X Bilinear Vs. 2X FSR on a 1080p display)"
+	"./zlabilinearbad.png"
+	"./zlafsrbad.png"
+>}}
+
+So, in short, if the rendering resolution is *higher* than the display’s resolution, FSR should be avoided.
+If the rendering resolution is *equal or lower* than the display’s resolution, FSR is an excellent shader based filtering option, but with the highest performance cost on the 
+list.
+
+As a way to cheat around this limitation, AMD and Nvidia Windows users can fake higher display resolutions with the use of 
+[Virtual Super Resolution](https://www.amd.com/en/support/kb/faq/dh2-010) and [Dynamic Super Resolution](https://www.nvidia.com/en-us/geforce/technologies/dsr/technology/) 
+respectively.
+This way, for example, a 1080p display can fake a 2160p (4K) maximum resolution, giving FSR a lot more information to play with.
+
+{{< single-title-imgs-compare
+	"The Legend of Zelda: Link's Awakening (2X Bilinear Vs. 2X FSR on a 4K display)"
+	"./zlabilineargood.png"
+	"./zlafsrgood.png"
+>}}
+
+Finally our current last option, `Anti-Aliasing Method`. 
+Pixels are almost squares, and squares don’t do a great job at displaying curves. Anti-aliasing methods are used to smooth out this undesired effect.
+At the moment we only offer [FXAA](https://en.wikipedia.org/wiki/Fast_approximate_anti-aliasing), which can help reduce the excessive sharpening shown when using FSR.
+In the near future, we plan to add more options like [MLAA/SMAA](https://en.wikipedia.org/wiki/Morphological_antialiasing)
+
+{{< single-title-imgs-compare
+	"Luigi's Mansion 3 (FSR Vs. FSR + FXAA, both on a 4K display)"
+	"./lm3fsr.png"
+	"./lm3fxaa.png"
+>}}
+
+Both the filters and anti-aliasing options can be changed while in-game using the buttons on the bottom left of yuzu’s window. 
+Changing the resolution multiplier in-game is not possible at the moment.
+
+As an additional note, A.R.T. also reimplements our `Anisotropic Filtering`!
+This option can be accessed from the Advanced tab in the graphics settings.
+In the past we recommended skipping this option as not all games are particularly fond of forcing values higher than they specified, but this new implementation is smarter, 
+avoiding previous drawbacks.
+
+At a low performance cost, it will increase the quality of textures at oblique angles, particularly ground textures, most noticeable in first and third person view games.
+It should now be safe to set at 16x and forget about it.
+
+{{< single-title-imgs-compare
+	"Pokémon Sword (default Anisotropic Filtering Vs. 16X)"
+	"./pksw1x.png"
+	"./pksw16x.png"
+>}}
+
+Last detail, AMD users should check that [Radeon Image Sharpening](https://www.youtube.com/watch?v=wTh_O9BZlGc) is disabled before using the scaler, as it can affect the 
```

Comment:
```suggestion
One last note, AMD users should ensure that [Radeon Image Sharpening](https://www.youtube.com/watch?v=wTh_O9BZlGc) is disabled before using the scaler, as it can affect the 
```